### ISSUE TITLE
Display Options for Message of the Day (MOTD)

### ISSUE THREAD

**Issue by YellowAdder** [Segs/Segs#752]
## Feature Summary
Provide an option to turn off the MOTD's display to players. 

### Proposed Implementation
A checkbox or other UI element in the server admin allowing for the MOTD to be disabled would be appropriate. 

### Alternatives and Additional Remarks
A slash command that can enable or disable the MOTD might be considered as well, or in addition to, an option in the admin interface. Additionally, if the MOTD's content/script is empty, its pop-up window should not display to players logging in.

**Comment by broxen** [comment]
Just a note: also add option to settings_template.cfg. We already have a motd timer from #658, so this seems a natural addition.

**Comment by nemerle** [comment]
I really think we should enable some lua hooks for events such as 'UserConnected'. This way the MOTD and similar things could be scripted?

**Comment by broxen** [comment]
Agree, plus anything else that might want to be done at user login. A pvp arena style server may always spawn players in a jail cell, for instance.
